### ISSUE TITLE
Feature request: skip if compressed content is larger than original

### ISSUE THREAD

**Issue by vHeemstra** [anncwb/vite-plugin-compression#35]
Add something like:
```js
if (size >= oldSize) {
  return
}
```

at:

https://github.com/vbenjs/vite-plugin-compression/blob/077eff37f45f12896326d687aa0e08f99b2b7a07/packages/core/src/index.ts#L99

To avoid deleting the original file if it will be skipped because of compressed size being larger, move these lines:

https://github.com/vbenjs/vite-plugin-compression/blob/077eff37f45f12896326d687aa0e08f99b2b7a07/packages/core/src/index.ts#L89-L91

after the suggested check code above.

See #36 for fix.

**Comment by nonzzz** [comment]
I think you can try [vite-plugin-compression2](https://github.com/nonzzz/vite-plugin-compression). Because this project seem die. I had to reimplement a version :(

**Comment by vHeemstra** [comment]
Thanks for the tip! Does `vite-plugin-compression2` support the option to skip the compressed version if it is larger than or equal the original version?

**Comment by nonzzz** [comment]
@vHeemstra  Of course. [size check](https://github.com/nonzzz/vite-plugin-compression/blob/master/src/index.ts#L119-L124)

By the way. `vite-plugin-compression2` has full unit test and e2e test. And it is consistent with the experience of the webpack version

**Comment by vHeemstra** [comment]
I switched to your updated package! It is indeed faster. :)
It works great, but I'm missing the ability to log the processed files (as this original package does).

I just made a PR adding the feature to skip compression if the compressed file size is larger than (or equal to) to original file size.

**Comment by nonzzz** [comment]
@vHeemstra  Ok. i will review your code
